models/userModel.js

```diff
@@ -11,7 +11,7 @@ const userSchema = new Schema(
 		email: { type: String, required: true, unique: true },
 		password: { type: String, required: true, minlength: 8 },
 		profilePhoto: { type: Buffer },
-		role: { type: String, enum: ['student', 'instructor', 'admin'], required: true },
+		role: { type: String, enum: ['student', 'instructor', 'admin'], required: false },
```

Comment:
why has it been set to false?

Reply:
in the signup routes the value isn't passed in the req and it's set as role: Role.Student or Role.Instructor

Reply:
Noted